gems/aws-sdk-s3/lib/aws-sdk-s3/plugins/streaming_retry.rb

```diff
@@ -84,8 +95,19 @@ def add_event_listeners(context, target)
             end
 
             context.http_response.on_error do |error|
-              if retryable_body?(context) && truncated_body?(error)
-                context.http_request.headers[:range] = "bytes=#{context.http_response.body.size}-"
+              puts context.http_response.body
```

Comment:
Is this going to stdout? Wasn't sure if this was intentional or left over from debugging

Reply:
Great catch, will remove!